app/src/main/java/com/infomaniak/swisstransfer/ui/components/HighlightedText.kt

```diff
@@ -86,13 +105,19 @@ private fun List<Rect>.transformForHighlightedStyle(
     verticalPadding: Float,
     horizontalPadding: Float,
     angleDegrees: Float,
+    @FloatRange(0.0, 1.0) progress: Float,
 ): Path = Path().apply {
     forEach { boundingBox ->
-        addPath(boundingBox.transformForHighlightedStyle(verticalPadding, horizontalPadding, angleDegrees))
+        addPath(boundingBox.transformForHighlightedStyle(verticalPadding, horizontalPadding, angleDegrees, progress))
     }
 }
 
-private fun Rect.transformForHighlightedStyle(verticalPadding: Float, horizontalPadding: Float, angleDegrees: Float): Path {
+private fun Rect.transformForHighlightedStyle(
+    verticalPadding: Float,
+    horizontalPadding: Float,
+    angleDegrees: Float,
```

Comment:
Is there a reason to pass `angleDegrees` as a `Float` and then convert it to a `Double` at the end ? Just convert your constant to a `Double` and you're good to go.

```kotlin
private const val ROTATION_ANGLE_DEGREE = -3.0
```